cmd/verify/verify.go

```diff
@@ -95,97 +95,97 @@ func Command() *cobra.Command {
 		0,
-		"if set, maximum number of rows to read per second during scanning per shard",
+		"If set, maximum number of rows to read per second during scanning per shard.",
 	)
 	cmd.PersistentFlags().BoolVar(
 		&verifyFixup,
 		"fixup",
 		false,
-		"whether to fix up any rows",
+		"Whether to fix up any rows.",
 	)
 	cmd.PersistentFlags().DurationVar(
 		&verifyContinuousPause,
 		"continuous-pause-between-runs",
 		0,
-		"time to pause between continuous runs",
+		"Time to pause between continuous runs.",
 	)
 	cmd.PersistentFlags().BoolVar(
 		&verifyContinuous,
 		"continuous",
 		false,
-		"whether verification should continuously run on each shard",
+		"Whether verification should continuously run on each shard.",
 	)
 	cmd.PersistentFlags().BoolVar(
 		&verifyLive,
 		"live",
 		false,
-		"enables live mode, which attempts to account for rows that can change in value by retrying them before marking them as an inconsistency",
+		"Enables live mode, which attempts to account for rows that can change in value by retrying them before marking them as an inconsistency.",
 	)
 	cmd.PersistentFlags().BoolVar(
 		&verifyRows,
 		"rows",
 		true,
-		"whether rows should be verified (otherwise, performs a more basic schema check)",
+		"Whether rows should be verified (otherwise, performs a more basic schema check).",
 	)
 	cmd.PersistentFlags().IntVar(
 		&verifyLiveVerificationSettings.RunsPerSecond,
 		"live-runs-per-second",
 		verifyLiveVerificationSettings.RunsPerSecond,
-		"if set, sets a maximum number of attempts to reverify per second",
+		"If set, sets a maximum number of attempts to re-verify per second.",
 	)
 	cmd.PersistentFlags().IntVar(
 		&verifyLiveVerificationSettings.MaxBatchSize,
 		"live-max-batch-size",
 		verifyLiveVerificationSettings.MaxBatchSize,
-		"maximum number of rows to retry at a time in live verification mode",
+		"Maximum number of rows to retry at a time in live verification mode.",
 	)
 	cmd.PersistentFlags().DurationVar(
 		&verifyLiveVerificationSettings.FlushInterval,
 		"live-flush-interval",
 		verifyLiveVerificationSettings.FlushInterval,
-		"maximum amount of time to wait before rows can start being retried",
+		"Maximum amount of time to wait before rows can start being retried.",
 	)
 	cmd.PersistentFlags().IntVar(
 		&verifyLiveVerificationSettings.RetrySettings.MaxRetries,
 		"live-retries-max-iterations",
 		verifyLiveVerificationSettings.RetrySettings.MaxRetries,
-		"maximum number of retries live reverification should run before marking rows as inconsistent",
+		"Maximum number of retries live re-verification should run before marking rows as inconsistent.",
 	)
 	cmd.PersistentFlags().DurationVar(
 		&verifyLiveVerificationSettings.RetrySettings.MaxBackoff,
 		"live-retry-max-backoff",
 		verifyLiveVerificationSettings.RetrySettings.MaxBackoff,
-		"maximum amount of time live reverification should take before retrying",
+		"Maximum amount of time live reverification should take before retrying.",
```

Comment:
Is this still accurate?

```suggestion
		"Maximum amount of time a row retry attempt should take before retrying again (live mode only).",
```